ovs/extensions/services/upstart.py

```diff
@@ -96,7 +96,7 @@ def get_service_status(name, client, return_output=False):
                     return status, output
                 return status
             # Normal cases - or if the above code didn't yield an outcome
-            if 'start' in output or 'is running' in output:
+            if 'start/running' in output or 'is running' in output:
                 if return_output is True:
                     return True, output
                 return True
```

Comment:
Shouldn't stop also be changed to stop/waiting ?
I don't know if there some sort of pre-stop for stopping a service